api/provider/providerRouter.js

```diff
@@ -173,7 +170,7 @@ router.get('/', function (req, res) {
  *        description: 'Profile not found'
  */
 
-router.get('/getServiceProviders', function (req, res) {
+router.get('/getServiceProviders', (req, res) => {
```

Comment:
Why no "next" on Line 173, and no .catch(next) at the end of the code block?  This seems to be the only changed instance in all the files that doesn't follow that pattern.

Reply:
Good catch @DomenicScarcella

Reply:
This one didn't have a .catch( err => ... ) so I didn't add it. I can change it though